glide.yaml

```diff
@@ -14,5 +14,11 @@ import:
   version: master
 - package: github.com/giantswarm/vaulthealthz
   version: master
+- package: github.com/giantswarm/vaultcrt
+  version: init
+- package: github.com/giantswarm/vaultpki
+  version: init
+- package: github.com/giantswarm/vaultrole
+  version: init
```

Comment:
These are the new dependencies. They are leaner and cleaner now. See also 

- https://github.com/giantswarm/vaultcrt/pull/1/files
- https://github.com/giantswarm/vaultpki/pull/1/files
- https://github.com/giantswarm/vaultrole/pull/1/files